cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -135,7 +135,7 @@ func TestAddFlags(t *testing.T) {
 				ConcurrentSATokenSyncs:                          10,
 				ConcurrentRCSyncs:                               10,
 				ConfigureCloudRoutes:                            false,
-				EnableContentionProfiling:                       true,
+				EnableContentionProfiling:                       false,
```

Comment:
is this change intentional?

Reply:
Yes. like `port` and `address`,  we should set it as value in `NewCloudControllerManagerOptions()`